src/CommitmentsV2/SFA.DAS.CommitmentsV2/Services/CohortDomainService.cs

```diff
@@ -537,9 +541,11 @@ private async Task ValidateOverlaps(DraftApprenticeshipDetails details, Cancella
 
         private async Task ValidateEmailOverlaps(DraftApprenticeshipDetails details, long? cohortId, CancellationToken cancellationToken)
         {
-            if (string.IsNullOrWhiteSpace(details.Email) || !details.StartDate.HasValue || !details.EndDate.HasValue) return;
+            var startDate = details.IsOnFlexiPaymentPilot.GetValueOrDefault() ? details.ActualStartDate : details.StartDate;
```

Comment:
Is it worth an extension method? something like GetStartDate() that resolves to the correct startDate if on flexi?

Reply:
sorted